src/components/MdField/MdFieldMixin.js

```diff
@@ -22,9 +22,8 @@ export default {
       },
       set (value) {
         if (value.constructor.toString().match(/function (\w*)/)[1].toLowerCase() !== 'inputevent') {
-          this.$nextTick(() => {
-            this.localValue = value
-          })
+          this.$emit('input', value)
+          this.localValue = value
```

Comment:
Hi, thanks for the post issue and PR.

Do you have any idea in this case without `nextTick`?

```vue
<template>
  <div>
    <md-field>
      <label>Initial Value</label>
      <md-input v-model="initial" @input="handleInput"></md-input>
    </md-field>
    <md-button @click="setValue">click</md-button>
    <div>result: {{inputValue}}</div>
  </div>
</template>

<script>
  export default {
    name: 'TextFields',
    data() {
      return {
        initial: '',
        inputValue: ''
      }
    },
    methods: {
      setValue() {
        this.initial = 'Set Value'
      },
      handleInput(val) {
        this.inputValue = val
      }
    }
  }
</script>
```

Steps:
  - type: 'abc'
  - click the button

before fix: 
```
result: Set Value
```

after fix:
```
result: 123
```